enhancements/network/openstack-dual-stack.md

```diff
@@ -0,0 +1,188 @@
+---
+title: openstack-dual-stack
+authors:
+  - "@maysamacedo"
+reviewers:
+  - "@mandre for SME review on OpenStack and general feedback"
+  - "@cybertron for SME review on on-premise networking"
+  - "@patrickdillon for SME review and approval on installer"
+approvers:
+  - "@mandre"
+api-approvers:
+  - "None"
+creation-date: 2023-03-13
+last-updated: 2023-03-13
+tracking-link:
+  - https://issues.redhat.com/browse/OCPBU-199
+  - https://issues.redhat.com/browse/OSASINFRA-1938
+---
+
+# OpenStack dual-stack
+
+Covers adding support of OpenShift dual-stack networking on OpenStack.
+
+## Summary
+
+Customers need support for OpenShift clusters running on OpenStack with IPv4 and IPv6.
+
+## Motivation
+
+Customers need dual-stack support to allow migration of workloads from IPv4 to IPv6
+as they are running out of IPv4 addresses. This will allow OpenShift on OpenStack to
+serve a wider variety of workloads.
+
+### User Stories
+
+* As an user of a IPI OpenShift cluster, I want to have ingress and egress working with dual-stack.
+* As an user of a IPI OpenShift cluster, I want clusterIP services to be working with dual-stack.
```

Comment:
This seems to be incompatible with the Non-Goal: Installer (IPI) handles creation...